gradle.properties

```diff
@@ -11,7 +11,7 @@ pluginUntilBuild = 243.*
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType = WS
-platformVersion = 2024.3
+platformVersion = 2024.3.3
```

Comment:
WebStorm 2024.3 seems disappeared from the repository (https://github.com/biomejs/biome-intellij/actions/runs/13496464754/job/37704771601). Updating to 2024.3.3 resolved the issue.